migration/sql-files/007-add-url-trailing-slash.sql

```diff
@@ -0,0 +1,6 @@
+-- add a trailing `/` when needed using POSIX regex to detect records to update
+update cluster set url = concat(url, '/') where url !~ '.*\/$';
+update cluster set console_url = concat(console_url, '/') where console_url !~ '.*\/$';
+update cluster set metrics_url = concat(metrics_url, '/') where metrics_url !~ '.*\/$';
+update cluster set logging_url = concat(logging_url, '/') where logging_url !~ '.*\/$';
+update cluster set logging_url = concat(logging_url, '/') where logging_url !~ '.*\/$';
```

Comment:
Duplicated logging_url normalization and missing app_dns.

Reply:
addressed in 90f7546